tests/benchdnn/rnn/rnn_aux.hpp

```diff
@@ -23,6 +23,7 @@
 
 namespace rnn {
 
+//NOLINTBEGIN(modernize-use-using)
```

Comment:
(nit) comment about GCC?

Reply:
Added this and supplied all other NOLINT places with relevant comments. Thanks.